packages/cms-cli/package.json

```diff
@@ -10,6 +10,7 @@
   "dependencies": {
     "@hubspot/cms-lib": "^2.2.1-beta.3",
     "chalk": "^4.1.0",
+    "dotenv": "^8.2.0",
```

Comment:
do we need to do anything to prevent `.env` files from being uploaded?

Reply:
hmm good point 🤔

Reply:
Added `.env` to the ignoreRules.